cmd/lukso/utils.go

```diff
@@ -192,6 +195,18 @@ func registerInputWithMessage(message string) (input string) {
 	return scanner.Text()
 }
 
+func registerSensitiveInputWithMessage(message string) (input string) {
+	fmt.Print(message)
+
+	byteSensitiveInput, err := term.ReadPassword(syscall.Stdin)
+	if err != nil {
```

Comment:
We don't handle error here outside of simple log, I think we should now whether we should continue with empty input (because 99% this will happen when error occurs)